respx/api.py

```diff
@@ -48,7 +54,9 @@ def pop(alias, default=...):
 def add(
     method: Union[str, Callable],
     url: Optional[Union[str, Pattern]] = None,
+    *,
     status_code: Optional[int] = None,
+    params: Optional[QueryParamTypes] = None,
```

Comment:
Since we're breaking the api with keyword-only args, I think the `params` kwarg should come first, since it belongs to the url/pattern.

It would look neater if response related kwargs, like `status_code`, lies next to each other.